src/databricks/labs/dqx/utils.py

```diff
@@ -53,6 +55,47 @@ def get_column_as_string(column: str | Column, normalize: bool = False) -> str:
     return col_str
 
 
+def is_valid_column_name(col_name: str) -> bool:
```

Comment:
Updates
- Added a new function called `is_valid_column_name` to check if an expression is a valid column expr when normalising the arguments